packages/neutrino-middleware-env/index.js

```diff
@@ -0,0 +1,5 @@
+const { EnvironmentPlugin } = require('webpack');
+
+module.exports = ({ config }, envs = []) => config
```

Comment:
At the moment, no, it's just whatever gets passed to the middleware.

https://github.com/mozilla-neutrino/neutrino-dev/pull/86/files#diff-ada11b9ab390b81e1edbc18574ff967aR30